HttpTrigger1/auth.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+from urllib.request import urlopen
+from jose import jwt
+
+AUTH0_DOMAIN = os.getenv("AUTH0_DOMAIN")
+API_AUDIENCE = os.getenv("API_AUDIENCE")
+ALGORITHMS = ["RS256"]
+PERMISSION = os.getenv("PERMISSION")
+
+
+class AuthError(Exception):
+    def __init__(self, message, status_code):
+        self.message = message
+        self.status_code = status_code
+
+
+def get_token_auth_header(req):
+    auth_header = req.headers.__http_headers__.get("authorization", None)
+    if auth_header is None:
+        raise AuthError("Authorization header is missing", 401)
+    header_parts = auth_header.split()
```

Comment:
Return a default value from the same type: `req.headers.__http_headers__["authorization"]` is a `str`
```suggestion
    auth_header = req.headers.__http_headers__.get("authorization", '')
    header_parts = auth_header.split()
    
    if auth_header == '':
        raise AuthError("Authorization header is missing", 401)
```

Reply:
Changed default value in case of missing header to `str()` instead of `None`